samples/go/csv/read.go

```diff
@@ -44,6 +46,31 @@ func KindsToStrings(kinds KindSlice) []string {
 	return strs
 }
 
+//EscapeStructFieldForCSV removes special characters and replaces spaces with camelCasing (camel' case turns to camelCase)
```

Comment:
Remove extra quote/apostrophe? `(camel' case turns to camelCase)` maybe `("camel case" becomes "camelCase")`?